data/json/npcs/godco/NPC_Chloe_Taylor_King.json

```diff
@@ -91,83 +13,155 @@
+    ],
     "responses": [
       {
-        "text": "Saving Sonia sounds like an interesting story; can you tell me what happened?",
-        "topic": "TALK_GODCO_Chloe_Sonia"
+        "text": "Aren't you a bit young to be carrying a knife?",
+        "effect": { "npc_add_var": "told_about_sonia", "type": "general", "context": "meeting", "value": "yes" },
+        "topic": "TALK_GODCO_chloe_knife"
+      },
+      {
+        "text": "Thanks for the info.  Let's chat about something else.",
+        "effect": { "npc_add_var": "told_about_sonia", "type": "general", "context": "meeting", "value": "yes" },
+        "topic": "TALK_GODCO_chloe_idle_chat"
       },
-      { "text": "Aren't you a bit too young to be carrying a knife?", "topic": "TALK_GODCO_Chloe_Knife" },
-      { "text": "What were you saying before?", "topic": "TALK_NONE" },
-      { "text": "I'd better get going.", "topic": "TALK_DONE" }
+      {
+        "text": "Thanks for the info.",
+        "effect": { "npc_add_var": "told_about_sonia", "type": "general", "context": "meeting", "value": "yes" },
+        "topic": "TALK_DONE"
+      }
     ]
   },
   {
+    "id": "TALK_GODCO_chloe_parents_where",
     "type": "talk_topic",
-    "id": "TALK_GODCO_Chloe_Sonia",
-    "dynamic_line": "I was home alone during the ongoing riots.  My parents chose the worst possible time to reconnect, which sucks, but I'm also kinda happy for them.  Anyway, they were part of the NECC people, so they knew about this place and their plans and all.  Thing is, it was, like, really far away.  I kinda didn't want to go alone but I also didn't want the rioters to eat me, so I asked Sonia - my neighbor - to evacuate with me, I guess.  So that was fun; escaping my hometown, walking down the country road, waiting for a horror to ambush us.  Spoiler alert: we were attacked.  So this zombie shambles in front of us and Sonia whacks it real hard with her guitar!  Its brain splatters all over the pavement, and it was super cool, but I also vomited because it was really disgusting.  We got here soon after that.  My parents weren't here, obviously.  Oh man, they must worry about me so much…",
+    "dynamic_line": [
+      "I don't know.  They were going to contract deal before the riots broke out, at one of military bases I think.  I don't know what happened to them from there."
```

Comment:
They were going to contract deal before the riots broke out, at one of military bases I think

This sentence needs something probably an action or something.  I'm not really sure.  but basically "contract deal" doesn't make sense in the sentence.

Reply:
They both worked for an arms manufacturer, probably RIVTEC. In my mind, I'd say they went there to show off a new weapons system before the army bought it, like the opening sequence from Iron Man (2008). 

Oh, now I see what you mean. I'll make it `a contract deal` instead.

Reply:
Just so you know, I've put all your suggestions on the main PR.